src/platform/macos/mod.rs

```diff
@@ -642,6 +642,10 @@ impl Window {
 
         Ok(())
     }
+
+    #[inline]
+    pub fn set_fullscreen_windowed(&self, fullscreen: bool) {
+    }
```

Comment:
I thought of that but it seemed to me it would make no sense to panic instead of noop for these things.